style.css

```diff
@@ -347,18 +403,167 @@ textarea {
 }
 
 .footer-email {
-  margin-top: -15px;
   font-style: normal;
   font-weight: normal;
   font-size: 20px;
-  line-height: 28px;
   text-align: center;
   color: #fff;
+  margin-top: 0;
 }
 
-.footer-section .footer-hr {
+.footer-hr {
   width: 50%;
   align-self: center;
   background-color: black;
   border: 2px solid black;
 }
+
+.work-socio {
+  display: flex;
+  width: 100%;
+  justify-content: space-around;
+  flex-direction: column;
+}
+
+.visitor-info,
+.visitor-email,
+.visitor-Message {
+  text-align: center;
+}
+
+.proj-2 .proj-img,
+.proj-4 .proj-img {
+  order: 9;
+}
+
+.img-2-4 {
+  order: 9;
+}
+
+.recnt-projects {
+  padding-top: 50px;
+}
+
+@media screen and (min-width: 850px) {
+  .proj-1,
```

Comment:
- [ ] Nice one implementing a media query for the desktop breakpoint. However, the required value for `min-width` property should be in range of `992px - 1100px`. This means that you can pick any value between `992px` and `1100px` as the `800px` you have is too small for a desktop breakpoint.

- [ ]  I can see you used only flexbox to layout elements on the page. Have you thought of using the [grid system](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids) to layout some of the elements? It looks pretty cool to use both flexbox and grid to layout elements so that you can have the knowledge of using both comfortably when displaying elements on the page. Please let's use `grid` to display at least one element on the page 👍